build.sbt

```diff
@@ -55,6 +54,5 @@ lazy val root = (project in file("."))
     )
   )
 
-lazy val vulkanSdk = System.getenv("VULKAN_SDK")
-javaOptions +=  s"-Dorg.lwjgl.vulkan.libname=$vulkanSdk/lib/libvulkan.1.dylib"
-
+// lazy val vulkanSdk = System.getenv("VULKAN_SDK")
```

Comment:
I think, in the, end, it really might have been unnecessary... Let's just remove it altogether and see if anything breaks.

Reply:
Got it.